repotracker/repotracker.go

```diff
@@ -184,7 +184,10 @@ func (repoTracker *RepoTracker) FetchRevisions(ctx context.Context) error {
 			return errors.WithStack(err)
 		}
 	}
-	ok, err := model.DoProjectActivation(ctx, projectRef.Id, time.Now())
+	// Fetch the most recent, non-ignored version (before the given time, but allowing for a 5 minute buffer).
+	// Do not use a version that is in the active cron range because those will be ignored when the one before
+	// it has an activate_at that is within the cron range.
+	ok, err := model.DoProjectActivation(ctx, projectRef.Id, time.Now().Add(-model.CronActiveRange))
```

Comment:
actually instead of this DoProjectActivation, I think we should be modifying the one [here](https://github.com/bynn/evergreen/blob/53ec3e9238b8fc6b1910ee1cd7c8a0e0a19535b4/repotracker/wrappers.go#L64-L65) instead. 

the call in this function, FetchRevisions, is called during the repotracker so I think this change might break repotracker into never activating the latest commit

Reply:
that's a good call, but I moved it into the versionActivationCatchup job itself instead where the time is passed into ActivateBuildsForProject

Reply:
I also tested this on staging by making sure that a new commit was picked up by the repotracker https://spruce-staging.corp.mongodb.com/version/sandbox_77dae8d5c9dfb67e9508c10deb4e643a15661cc7/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC